LT-ScriptExport.ps1

```diff
@@ -1090,73 +1093,73 @@ if(Test-Path $LTShareSource){
 
 $SQLQuery = "select table_name from tables where table_type = 'BASE TABLE' and table_schema = '$MySqlDatabase'"
 $rows = Get-LTData $SQLQuery -info_schema
 
 foreach($row in $rows.table_name){
-    $filename = $row
+    $filename = [System.IO.Path]::Combine($BackupPath, $row)
     $createCol = "Create Table"
     $SQLQuery = "SHOW CREATE TABLE $MySqlDataBase.$row"
     ## silent continue due to certain tables failing to export config
     ## replace the auto_increment field to have sane diffs
-    (Get-LTData $SQLQuery -info_schema -ErrorAction SilentlyContinue).$createCol -replace ' AUTO_INCREMENT=[0-9]*\b','' | Out-File -Force "$row.sql"
+    (Get-LTData $SQLQuery -info_schema -ErrorAction SilentlyContinue).$createCol -replace ' AUTO_INCREMENT=[0-9]*\b','' | Out-File -Force "$filename.sql"
 }
 
-New-BackupPath "DB\procedures"
+$BackupPath = New-BackupPath "DB\procedures"
 
 $SQLQuery = "SHOW PROCEDURE STATUS WHERE db = '$MySqlDatabase'"
 $rows = Get-LTData $SQLQuery -info_schema
 
 foreach($row in $rows.name){
-    $filename = $row
+    $filename = [System.IO.Path]::Combine($BackupPath, $row)
     $createCol = "Create Procedure"
     $SQLQuery = "SHOW CREATE PROCEDURE $MySqlDataBase.$row"
     ## silent continue due to certain tables failing to export config
     ## replace the auto_increment field to have sane diffs
-    (Get-LTData $SQLQuery -info_schema -ErrorAction SilentlyContinue).$createCol -replace ' AUTO_INCREMENT=[0-9]*\b','' | Out-File -Force "$row.sql"
+    (Get-LTData $SQLQuery -info_schema -ErrorAction SilentlyContinue).$createCol -replace ' AUTO_INCREMENT=[0-9]*\b','' | Out-File -Force "$filename.sql"
 }
 
 
-New-BackupPath "DB\functions"
+$BackupPath = New-BackupPath "DB\functions"
 
 $SQLQuery = "SHOW FUNCTION STATUS WHERE db = '$MySqlDatabase'"
 $rows = Get-LTData $SQLQuery -info_schema
 
 foreach($row in $rows.name){
-    $filename = $row
+    $filename = [System.IO.Path]::Combine($BackupPath, $row)
     $createCol = "Create Function"
     $SQLQuery = "SHOW CREATE FUNCTION $MySqlDataBase.$row"
     ## silent continue due to certain tables failing to export config
     ## replace the auto_increment field to have sane diffs
-    (Get-LTData $SQLQuery -info_schema -ErrorAction SilentlyContinue).$createCol -replace ' AUTO_INCREMENT=[0-9]*\b','' | Out-File -Force "$row.sql"
+    (Get-LTData $SQLQuery -info_schema -ErrorAction SilentlyContinue).$createCol -replace ' AUTO_INCREMENT=[0-9]*\b','' | Out-File -Force "$filename.sql"
 }
 
-New-BackupPath "DB\events"
+$BackupPath = New-BackupPath "DB\events"
 
 $SQLQuery = "SHOW EVENTS WHERE db = '$MySqlDatabase'"
-$rows = Get-LTData $SQLQuery -info_schema
+$rows = Get-LTData $SQLQuery 
 
 foreach($row in $rows.name){
-    $filename = $row
-    $createCol = "Create Events"
-    $SQLQuery = "SHOW CREATE EVENTS $MySqlDataBase.$row"
+    $filename = [System.IO.Path]::Combine($BackupPath, $row)
+    $createCol = "Create Event"
+    $SQLQuery = "SHOW CREATE EVENT $MySqlDataBase.$row"
```

Comment:
Ha whoops. I don't have any events in my db, so i couldn't test this section

Reply:
I didn't even know you could do it the way you tried until I saw your code.